HelpMyStreetFE/HelpMyStreetFE/Helpers/JobStatusHelpers.cs

```diff
@@ -81,6 +81,8 @@ public static int UsualOrderOfProgression(this JobStatuses jobStatus)
                 JobStatuses.InProgress => 40,
                 JobStatuses.Done => 50,
                 JobStatuses.Cancelled => 100,
+                JobStatuses.Approved => 200,
+                JobStatuses.Rejected => 300,
```

Comment:
Maybe put Rejected at 15, so that if Rejected and Open have the same timestamp then Rejected will be shown first?

Reply:
Resolved in branch